helm/deploy_test.go

```diff
@@ -132,6 +137,21 @@ func TestHelmDeployConfigureUndeploy(t *testing.T) {
 
 	// Check that the rendered helm env file works
 	validateHelmEnvFile(t, testServiceAccountKubectlOptions)
+
+	// Revoke the tiller service account
+	rbacGroups := []string{}
+	rbacUsers := []string{}
+	serviceAccounts := []string{fmt.Sprintf("%s/%s", namespaceName, testServiceAccountName)}
+	require.NoError(t, RevokeAccess(kubectlOptions, namespaceName, rbacGroups, rbacUsers, serviceAccounts))
+
+	// No role for service account
+	assert.Error(t, validateNoRole(t, kubeClient, namespaceName, testServiceAccountName))
+
+	// No rolebinding for service account
+	assert.Error(t, validateNoRoleBinding(t, kubeClient, namespaceName, testServiceAccountName))
+
+	// No TLS keypair secret for service account
+	assert.Error(t, validateNoTLSSecret(t, kubectlOptions, namespaceName, serviceAccountName))
```

Comment:
+1 on Josh's comment above: we should check *which* error comes back in all these checks, as these could be returning errors for a variety of reasons, and not all of them are necessarily what we're checking for.

Reply:
Also, is the _real_ way to test this to try to make requests as the user, check they succeed, revoke the user's access, and check the requests again to make sure they fail?

Reply:
Yes, that would certainly be the more complete test. However, `kubergrunt` currently doesn't run any tests from the perspective of multiple users. It relies on the kubectl credentials currently available in the executing user's path. Adding the ability to switch from admin/user context is a bit more time than I can afford at the moment unfortunately and would have some downstream implications.